src/main/java/uk/gov/hmcts/reform/sandl/snlevents/repository/queries/SessionQueries.java

```diff
@@ -21,7 +21,7 @@
         + "  r.room_type_code                       AS room_type_code, "
         + "  rt.description                         AS room_description, "
         + "  (SELECT COUNT(hearing_part.id) FROM hearing_part "
-        + "   WHERE hearing_part.session_id = s.id) AS hearing_parts_count, "
+        + "   WHERE hearing_part.session_id = s.id AND hearing_part.is_deleted = false) AS hearing_parts_count, "
```

Comment:
Its very small thing but can you change 'false' to 'FALSE' (using capitals characters)